meinberlin/apps/budgeting/assets/VoteButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import django from 'django'
+import CheckboxButton from '../../contrib/assets/CheckboxButton'
+
+export const VoteButton = (props) => {
```

Comment:
I guess you have to wrap it to use as a widget?
Would it make sense to use the `VoteButton` for the list item as well?

Reply:
yes it would